README.md

```diff
@@ -143,13 +148,66 @@ The image will validate that your SPT version in the serverfiles matches the ima
+
+```
+https://github.com/project-fika/Fika-Plugin/releases/download/v0.9.9015.15435/Fika.Release.0.9.9015.15435.zip
+https://github.com/Solarint/SAIN/releases/download/v3.1.0-Release/SAIN-3.1.0-Release.7z https://github.com/DrakiaXYZ/SPT-BigBrain/releases/download/1.0.1/DrakiaXYZ-BigBrain-1.0.1.7z
+https://github.com/DrakiaXYZ/SPT-Waypoints/releases/download/1.5.1/DrakiaXYZ-Waypoints-1.5.1.7z
+https://github.com/Nympfonic/UnityToolkit/releases/download/v1.0.1/UnityToolkit-1.0.1.7z
+https://github.com/Skwizzy/SPT-LootingBots/releases/download/v1.3.5-spt-3.9.0/Skwizzy-LootingBots-1.3.5.zip
+https://github.com/dwesterwick/SPTQuestingBots/releases/download/0.7.0/DanW-SPTQuestingBots.zip https://github.com/mpstark/SPT-DynamicMaps/releases/download/0.3.4/DynamicMaps-0.3.4-b6d8bf85.zip
+
+```
+#### MOD_URLS_TO_DOWNLOAD Environment Variable
+
+Just Set the environment variable `MOD_URLS_TO_DOWNLOAD` to a list of the URLs you want it to download. I don't think you can use new lines in environment variables, so just stick to spaces, but otherwise it would be the same as the `mod_urls_to_download.txt` example above.
+
+> [!WARNING] If you use both methods, it may conflict if you download multiple versions of the same mod at the same time.
+
+The URLs should point to a direct file download of a `.zip`, `.7z`, `.tar/.tar.gz`, or `.dll` file. It is assumed that compressed downloads (`.zip`, `.7z`, and `.tar/.tar.gz`) are already organized into the `BepInEx/plugins` and/or `user/mods` directory(ies). Most mod developers do this but not all of them. If a mod is not correctly organized then it will still be downloaded and extracted but the files will be moved to the `mods_download/remains` directory for you to handle manually.
+
+If a mod requires any post-installation configuration, you will still need to to this manually.
+
+A few other notes
+* You can add or change the URLs whenever you want. Any new URLs will be downloaded and installed the next time the container is restarted.
+* Removing a URL from the specified URLs does not remove the downloaded mod/files, it only stops it from checking that URL again.
+* If you want to redownload the same url, you will need to manually remove it from `mods_download/mod_urls_downloaded.txt` file.
+
+### Mod updates
+When a mod is updated, you will need to add the new URL using one of the methods above. It will be downloaded, extracted, and then merged, overwiriting any conflicting files in the installation. For simple mods that is probably enough. If the mod developer states that you will need to uninstall a previous version to update, you will have to do this manually. You may do that at any time if you want to be extra cautious.
```

Comment:
```suggestion
When a mod is updated, you will need to add the new URL using one of the methods above. It will be downloaded, extracted, and then merged, overwriting any conflicting files in the installation. For simple mods that is probably enough. If the mod developer states that you will need to uninstall a previous version to update, you will have to do this manually. You may do that at any time if you want to be extra cautious.
```